addons/payment/static/src/xml/payment_post_processing.xml

```diff
@@ -36,7 +36,7 @@
                 <h5 class="alert-heading mb-0" t-out="alert_header ? alert_header : 'Please wait...'"/>
                 <p t-if="alert_message" class="mb-0" t-out="alert_message"/>
             </div>
-            <a t-att-href="landing_route" class="alert-link ms-auto text-nowrap">Skip <i class="oi oi-arrow-right small"/></a>
+            <a t-att-href="landing_route" class="alert-link ms-auto text-nowrap">Skip<i class="oi oi-arrow-right small"/></a>
```

Comment:
Hello! Thank you for your fixes, I just have a remark regarding the arrow: 

You removed the space, but the arrow is still there (maybe the icon disappeared on your screenshot  because of the browser cache? This arrow character is new so it's possible that you still have the old font cached)

We have to either keep the space and the arrow, or remove the space and the `<i class="oi oi-arrow-right small"/>` I'm visually fine with both, it just depends on how much we want the skip to be noticeable.

Reply:
As dala said we will keep arrow, so I am adding back space.